### PR TITLE
Make style improvements

### PR DESCRIPTION
**Pull request by frontsideair**
Sorry this took so long but I made my style improvements not depend on react-bootstrap, you can review and merge if you want.

### CODE REVIEW

**Comment by AllenFang** [comment]
hi @frontsideair, I'll review it. thanks u very much  :+1:

**Comment by frontsideair** [comment]
I added one more commit to fix some bugs.

File: src/toolbar/ToolBar.js [hunk 1]
```diff
@@ -119,7 +119,9 @@ class ToolBar extends React.Component{
             <i className="glyphicon glyphicon-trash"></i> Delete
           </button>:null;
     var searchTextInput = this.props.enableSearch?
-      <input type='text' placeholder={this.props.searchPlaceholder?this.props.searchPlaceholder:'Search'} onKeyUp={this.handleKeyUp.bind(this)}/>:null;
+      <div className="form-group form-group-sm">
+        <input className="form-control" type='text' placeholder={this.props.searchPlaceholder?this.props.searchPlaceholder:'Search'} onKeyUp={this.handleKeyUp.bind(this)}/>
```
Comment (imaireaux) [inline src/toolbar/ToolBar.js]: Hi,

I am currently searching for a way to customize some elements like the search input.
I can see that you add the `form-control` class to it.
May be you could make it customizable in this PR ?

Something like : 
`<input className={this.props.searchClassName?this.props.searchClassName:'form-control'} type='text' placeholder={this.props.searchPlaceholder?this.props.searchPlaceholder:'Search'} onKeyUp={this.handleKeyUp.bind(this)}/>`

Comment (AllenFang) [inline src/toolbar/ToolBar.js]: @imaireaux, id you want to use default search element in <code>react-bootstrap-table</code>, you can disable <code>search</code> and customize yours and call <code>handleSearch</code> on <code>BootstrapTable</code>, check the example on following:
```js

function search(text) {
    this.refs.table.handleSearch(text);
}

<BootstrapTable ref='table' data={products}>
    //...
</BootstrapTable>
```
See the <code>handleSearch</code> [docs](http://allenfang.github.io/react-bootstrap-table/docs.html)

**Comment by AllenFang** [comment]
Nice improvement :)  I'll merge it on next version. Thanks

**Comment by AllenFang** [comment]
Released on <code>v1.4.0</code>, thanks again:)

**Comment by frontsideair** [comment]
You're very welcome

File: src/toolbar/ToolBar.js (hunk 1 above)
Comment (frontsideair) [inline src/toolbar/ToolBar.js]: @imaireaux I understand your expectation but I'm afraid it'd bloat the props if we passed this class name to the component. The more general solution to your problem would be to add a class to the main container and write a CSS selector to affect search input specifically. Something like this:

```
.react-bs-container.my-table .tool-bar .form-group input {
  color: red;
}
```

Comment (imaireaux) [inline src/toolbar/ToolBar.js]: You're both right. Before this PR, this input had no bootstrap class, but this is not the case anymore.
And it's true there are other ways to customize this input.

Comment (frontsideair) [inline src/toolbar/ToolBar.js]: You're welcome.

**Pull request merged**